### ISSUE TITLE
false positive when breaking up nested dictionary access

### ISSUE THREAD

**Issue by tponthieux** [PyCQA/pycodestyle#191]
http://stackoverflow.com/questions/16204076/pep8-compliant-deep-dictionary-access
```python
dct = {
    'long_key_name_one': {
        'long_key_name_two': {
            'long_key_name_three': {
                'long_key_name_four': {
                    'long_key_name_five': 1
                }
            }
        }
    }
}
```

Why is this not pep8 compliant?
```python
dct['long_key_name_one']['long_key_name_two']\
    ['long_key_name_three']['long_key_name_four']\
    ['long_key_name_five'] = 2
```
Do you really want to force the world to do this...
```python
dct['long_key_name_one']['long_key_name_two'][
    'long_key_name_three'
]['long_key_name_four']['long_key_name_five'] = 2
```
or this...
```python
(dct['long_key_name_one']['long_key_name_two']
       ['long_key_name_three']['long_key_name_four']
       ['long_key_name_five']) = 2
```
